app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_like_count
+    post.increment!(:likes_counter)
+  end
```

Comment:
- Good job done so far, but according to the project requirements, I suggest you call this function after the data has been saved to your database. Please fix this 🙏 i.e `after_save :update_like_count`